io.openems.edge.levl.controller/src/io/openems/edge/levl/controller/LevlControlRequest.java

```diff
@@ -0,0 +1,140 @@
+package io.openems.edge.levl.controller;
+
+import com.google.gson.JsonObject;
+import io.openems.common.exceptions.OpenemsError;
+import io.openems.common.exceptions.OpenemsError.OpenemsNamedException;
+import io.openems.common.jsonrpc.base.JsonrpcRequest;
+import io.openems.common.utils.JsonUtils;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class LevlControlRequest {
```

Comment:
Would be good to put a version on this request, so it's easier to change the API version eventually

Reply:
You're right. It's definitely something we'll add at a later date. For the beta it will be fine.

Reply:
I'm okay with this approach, but it eases it if it's built in from the beginning